templates/fine-tune-llm_v2/training_configs/custom/meta-llama/Meta-Llama-3-8B/lora/4xA10-512-mlflow.yaml

```diff
@@ -0,0 +1,67 @@
+# Change this to the model you want to fine-tune
+model_id: meta-llama/Meta-Llama-3-8B-Instruct
+
+# Change this to the path to your training data
+train_path: s3://air-example-data/gsm8k/train.jsonl
+
+# Change this to the path to your validation data. This is optional
+valid_path: s3://air-example-data/gsm8k/test.jsonl
+
+# Change this to the context length you want to use. Examples with longer
+# context length will be truncated.
+context_length: 512
+
+# Change this to total number of GPUs that you want to use
+num_devices: 4
+
+# Change this to the number of epochs that you want to train for
+num_epochs: 3
+
+# Change this to the batch size that you want to use
+train_batch_size_per_device: 4
+eval_batch_size_per_device: 4
+
+# Change this to the learning rate that you want to use
+learning_rate: 1e-4
+
+# This will pad batches to the longest sequence. Use "max_length" when profiling to profile the worst case.
+padding: "longest"
+
+# By default, we will keep the best checkpoint. You can change this to keep more checkpoints.
+num_checkpoints_to_keep: 1
+
+# Deepspeed configuration, you can provide your own deepspeed setup
+deepspeed:
+  config_path: deepspeed_configs/zero_3_offload_optim+param.json
+
+logger:
+  provider: mlflow
+  provider_config:
+    tracking_uri: "https://*******.cloud.databricks.com"
+    experiment_name: "/Users/*******/llmforge"
+    create_experiment_if_not_exists: True
```

Comment:
Hmm in general, we've always made configs that are runnable by default withexample datasets, etc. I guesss this is unavoidable for MlFLow given that the tracking uri is internal to their MLFlow account? 

In any case we should document that this needs to change

```suggestion
# Logging config
logger:
  provider: mlflow
  provider_config:
     # Make sure to change this to your tracking uri! 
    tracking_uri: "https://*******.cloud.databricks.com"
    # Make sure to change this to the path where experiments should be logged!
    experiment_name: "/Users/*******/llmforge"
    create_experiment_if_not_exists: True
```